src/ontology/cl-edit.owl

```diff
@@ -15507,7 +15507,9 @@ AnnotationAssertion(oboInOwl:creation_date obo:CL_0002205 "2010-09-02T02:38:34Z"
 AnnotationAssertion(oboInOwl:hasAlternativeId obo:CL_0002205 "CL:1000386"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0002205 "FMA:263220"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0002205 "brush cell of lobular bronchiole"^^xsd:string)
-EquivalentClasses(obo:CL_0002205 ObjectIntersectionOf(obo:CL_0002075 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002051)))
+EquivalentClasses(obo:CL_0002205 ObjectIntersectionOf(obo:CL_0002204 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002051)))
+SubClassOf(obo:CL_0002205 obo:CL_0002075)
+SubClassOf(obo:CL_0002205 obo:CL_0002208)
```

Comment:
'bronchial epithelial cell' is defined as an epithelial cell that is part of bronchus. Bronchus and bronchiole are different anatomical structures, so consider removing this.

Reply:
Thank you for the comment, I will eliminate it

Reply:
'brush cell of bronchus' is still an inferred parent class. The reason being that 'bronchiole' is 'branching part of' 'bronchus', and 'branching part of' is a subproperty of part of. 
<img width="1346" alt="Screenshot 2023-02-06 at 14 38 26" src="https://user-images.githubusercontent.com/94959119/217000624-c4dfb5ef-ff22-4e46-b1fb-ae6371318935.png">

Reply:
Definition of 'branching part of': x is a branching part of y if and only if x is part of y and x is connected directly or indirectly to the main stem of y

A bronchiole is not part of a bronchus (they are separate), so I don't think that property is correctly applied in this case. Asserting that a bronchiole is a 'branching apart of' a 'tracheobronchial tree' seems more appropriate.